assets/src/dashboard/app/views/shared/savedTemplateGridView.js

```diff
@@ -22,7 +22,7 @@ import { useRef, useEffect, useState, useCallback } from 'react';
 import { getRelativeDisplayDate } from '@web-stories-wp/date';
 import { __, sprintf } from '@web-stories-wp/i18n';
 import { trackEvent } from '@web-stories-wp/tracking';
-
```

Comment:
Is there a reason to remove the extra line between the import and the starting comment? Looks like it's done everywhere in the files. Isn't it more readable when there's an empty line between the import and the next comment?

Reply:
@spacedmonkey This comment was marked resolved but I can still see various places where there is no empty line between the import and the comment block. Thoughts?

(Marked it unresolved until then)

Reply:
Fixed now.